cmd/kubeadm/app/cmd/init.go

```diff
@@ -191,22 +191,22 @@ func AddInitConfigFlags(flagSet *flag.FlagSet, cfg *kubeadmapiext.MasterConfigur
 func AddInitOtherFlags(flagSet *flag.FlagSet, cfgPath *string, skipPreFlight, skipTokenPrint, dryRun *bool) {
 	flagSet.StringVar(
 		cfgPath, "config", *cfgPath,
-		"Path to kubeadm config file (WARNING: Usage of a configuration file is experimental)",
+		"Path to kubeadm config file (WARNING: Usage of a configuration file is experimental).",
```

Comment:
You dropped the parentheses for this same message in `cmd/kubeadm/app/cmd/config.go:130`, can you drop them here too for consistency?

Reply:
will update in a minute. let me fire up my build setup...